src/core/DrawingEngineCore.ts

```
@@ -0,0 +1,669 @@
+
+    this.wasRubberBanding = true
+    setTimeout(() => { this.wasRubberBanding = false }, 50)
+
+    const topLeft: [number, number] = [Math.min(startPoint.x, e.point.x), Math.min(startPoint.y, e.point.y)]
+    const bottomRight: [number, number] = [Math.max(startPoint.x, e.point.x), Math.max(startPoint.y, e.point.y)]
+    const hits = this.map.queryRenderedFeatures([topLeft, bottomRight], { layers: CLICKABLE_LAYERS })
+    const ids = new Set(hits.map((f) => f.properties?._id as string).filter(Boolean))
+
+    if (e.originalEvent.shiftKey) {
+      const next = new Set(this._selectedFeatureIds)
+      ids.forEach((id) => next.add(id))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)